hack/check_error_log_msg_format.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+source "hack/lib/test_lib.sh"
+
+echo "Checking format of error and log messages..."
+allfiles=$(listFiles)
+log_case_output=$(grep -ERn '(Error\((.*[Ee]rr|nil), |[^(fmt\.)]Error(f)?\(|Fatal(f)?\(|Info(f)?\(|Warn(f)?\()"[[:lower:]]' $allfiles | sort -u)
+if [ -n "${log_case_output}" ]; then
+  echo -e "Log messages in wrong case:\n${log_case_output}"
```

Comment:
Can we say what the right case should be.
```suggestion
  echo -e "Log messages should begin with upper case:\n${log_case_output}"
```

Same for the other two error messages below.